### DISCUSSION TITLE
is it possible to create Android version ?

### DISCUSSION THREAD

**Discussion by izac3d** [huxingyi/dust3d#139] [Ideas]
because android tablets has touch screen and we can use a pen for modeling faster.

**Comment by ghost** [comment]
I would love to use Dust3d android app. My phone is 6GB of lpddr4 ram and 64GB of ssd which is fast. On the other hand, my pc has only 2GB of ddr3 ram and 500GB of hdd hard drive which takes nearly 3 minutes to load ubuntu 22.04. It's so slow to work with potato pc. Oh man, that will be a bit hit on playstore🎯.

**Comment by huxingyi** [comment]
It's possible since I have removed most of the external dependecies, but still lots of work to do to make it run on non-desktop environment.
